lua/cyberdream/extensions/neogit.lua

```diff
@@ -0,0 +1,36 @@
+local M = {}
+
+--- Get extension configuration
+--- @param opts Config
+--- @param t CyberdreamPalette
+function M.get(opts, t)
+    opts = opts or {}
+    local highlights = {
+        -- neogit
+        NeogitNormal               = { bg = "NONE" },
```

Comment:
Can this be linked to the `Normal` highlight group instead? This will be a bit more robust for both transparent and opaque backgrounds.

Reply:
Should be all set, thanks!